From 19482794a621d542b223219940e836257d4ae2c9 Mon Sep 17 00:00:00 2001
From: Jan Tojnar <jtojnar@gmail.com>
Date: Thu, 11 Jun 2020 15:31:09 +0200
Subject: [PATCH] build: Declare public dependencies in pkg-config

gdk-pixbuf and X11 headers are included in public headers so they should be in Requires.
---
 gdk-pixbuf-xlib/meson.build | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/gdk-pixbuf-xlib/meson.build b/gdk-pixbuf-xlib/meson.build
index a19476d..4e34f8c 100644
--- a/gdk-pixbuf-xlib/meson.build
+++ b/gdk-pixbuf-xlib/meson.build
@@ -44,5 +44,9 @@ pkgconfig.generate(
   name: 'GdkPixbuf Xlib',
   description: 'GdkPixbuf rendering for Xlib',
   filebase: 'gdk-pixbuf-xlib-2.0',
+  requires: [
+    gdk_pixbuf_dep,
+    x11_dep,
+  ],
   subdirs: api_name,
 )
-- 
GitLab

